packages/txstream/chopper/chopper.go

```diff
@@ -13,10 +13,11 @@ import (
 
 const (
 	// for the final data packet to be not bigger than tangle.MaxMessageSize
-	// 4 - chunk id, 1 seq nr, 1 num chunks, 2 - data len
-	chunkHeaderSize     = 4 + 1 + 1 + 2
+	// 4 - chunk id, 1 seq nr, 2 num chunks, 2 - data len
+	chunkHeaderSize     = 4 + 1 + 2 + 2
 	maxTTL              = 5 * time.Minute
 	cleanupLoopInterval = 10 * time.Second
+	maxNChunks          = 2200
```

Comment:
Why not 2^16?

Reply:
I was aiming for a max size of ~1MB. But we can indeed use all the capacity. updating it 👍